pkgs/applications/misc/metamorphose2/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchgit, makeWrapper, gettext
+, python3, python3Packages
+}:
+
+stdenv.mkDerivation {
+  pname = "metamorphose2";
+  version = "0.10.0beta";
+
+  # exif-py vendored via submodule
+  # mutagen vendored via copy
+  src = fetchgit {
```

Comment:
Please use fetchFromGitHub

Can we devendor the vendored things?

Reply:
I didn't have much luck with exif-py (the exifread package). The code uses "from exif import EXIF" while the docs instruct to use "import exifread" so I'm guessing the api was changed which is why it's vendored to a two years old version. It comes with an cli EXIF.py so that probably was the old entry point for both the library functions and the command interface...
Anyhow, I did manage to devendeor mutagen I think. well, it doesn't fail to import at least... Haven't tested if it works though since I have no idea what it supposed to do :D

Reply:
Vendoring at its finest.

We should still use fetchFromGitHub with fetchSubmodule = true.

Reply:
k